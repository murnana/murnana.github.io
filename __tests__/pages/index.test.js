import React from "react"
import renderer from "react-test-renderer"
import IndexPage from "../../src/pages/index"

describe("IndexPage", () => {
  it("render", () => {
    const data = {
      site: {
        siteMetadata: {
          title: `site.siteMetadata.title`,
        },
      },
    }

    const tree = renderer.create(<IndexPage data={data} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
