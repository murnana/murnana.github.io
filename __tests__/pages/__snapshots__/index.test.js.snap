// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IndexPage render 1`] = `
<div>
  <h1>
    site.siteMetadata.title
  </h1>
  <img
    alt="Gravatar"
    src="https://www.gravatar.com/avatar/b08382f69497236d00e369d103e2a535?s=200"
  />
  <ul>
    <li>
      <a
        href="/hobby"
      >
        Hobby
      </a>
    </li>
  </ul>
</div>
`;
